content/community/moderation.md

```diff
@@ -62,10 +62,13 @@ Moderators need to stick to the [Codewars Code of Conduct](/community/rules/), t
 
 They can be recognized by the symbol of a shield (🛡️) displayed next to their username.
 
+
 ## How to get in touch
 
-All kinds of issues are to be reported on [GitHub](https://github.com/codewars/codewars.com/issues) using an appropriate issue template if available. There are already issues dedicated to handling [spammers](https://github.com/codewars/codewars.com/issues/1493) and [cheaters](https://github.com/codewars/codewars.com/issues/1378).
+Many members of the moderation team (but not all of them) can be reached via [Gitter](https://gitter.im/Codewars/codewars.com). Message them directly or post on one of the public channels if you need their assistance.
```

Comment:
All of the community team has Gitter...

Reply:
Today, yes. But in general some might not have, or might be not active on Gitter.

Reply:
```suggestion
Many members of the moderation team can be reached via [Gitter](https://gitter.im/Codewars/codewars.com). Message them directly or post on one of the public channels if you need their assistance.
```